azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/robots/SetupScreenRobot.kt

```diff
@@ -43,6 +51,26 @@ class SetupScreenRobot : ScreenRobot<SetupScreenRobot>() {
         return this
     }
 
+    fun verifyIsJoinCallButtonDisabled(): SetupScreenRobot {
```

Comment:
add the call to wait until   `R.id.azure_communication_ui_setup_join_call_button` is displayed